Building master fails on linux

Hi,

As of writing, master doesn't build on linux.
I'm using Swift 5.1.4 on Fedora 31

Here's the error:

```
/home/arnaud/dev/playground/vapor_toolbox2/Sources/VaporToolbox/exec.swift:36:42: error: value of optional type 'UnsafeMutablePointer<Int8>?' must be unwrapped to a value of type 'UnsafeMutablePointer<Int8>'
    let spawned = posix_spawnp(&pid, argv[0], &fileActions, nil, argv + [nil], envp + [nil])
                                         ^
/home/arnaud/dev/playground/vapor_toolbox2/Sources/VaporToolbox/exec.swift:36:42: note: coalesce using '??' to provide a default when the optional value contains 'nil'
    let spawned = posix_spawnp(&pid, argv[0], &fileActions, nil, argv + [nil], envp + [nil])
                                         ^
                                             ?? <#default value#>
/home/arnaud/dev/playground/vapor_toolbox2/Sources/VaporToolbox/exec.swift:36:42: note: force-unwrap using '!' to abort execution if the optional value contains 'nil'
    let spawned = posix_spawnp(&pid, argv[0], &fileActions, nil, argv + [nil], envp + [nil])
                                         ^
                                            !
```

Adding a ! to argv[0] works around it.

I haven't gotten around testing this on macOS yet, maybe argv is different.

--- a/.github/FUNDING.yml
+++ b/.github/FUNDING.yml
@@ -1,0 +1,2 @@
+github: [tanner0101] # loganwright, joscdk
+open_collective: vapor

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,0 +1,18 @@
+name: test
+on:
+- pull_request
+jobs:
+  toolbox_xenial:
+    container: 
+      image: vapor/swift:5.2-xenial
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: swift build --enable-test-discovery --sanitize=thread
+  toolbox_bionic:
+    container: 
+      image: vapor/swift:5.2-bionic
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - run: swift build --enable-test-discovery --sanitize=thread

--- a/Package.resolved
+++ b/Package.resolved
@@ -5,18 +5,18 @@
         "package": "console-kit",
         "repositoryURL": "https://github.com/vapor/console-kit.git",
         "state": {
-          "branch": "master",
-          "revision": "535874e4654b17cebb422b9e17353e85d421a118",
-          "version": null
+          "branch": null,
+          "revision": "de99d83c9f3c298371002729cc0601e4b5d9e0c9",
+          "version": "4.0.0-rc.1"
         }
       },
       {
         "package": "Mustache",
         "repositoryURL": "https://github.com/tanner0101/mustache.git",
         "state": {
-          "branch": "master",
+          "branch": null,
           "revision": "2169b3a2ba3add4957faaa1bdaa0b7b1e0013b08",
-          "version": null
+          "version": "0.1.0"
         }
       },
       {
@@ -32,18 +32,18 @@
         "package": "swift-nio",
         "repositoryURL": "https://github.com/apple/swift-nio.git",
         "state": {
-          "branch": "master",
-          "revision": "ea5a64978d944567498533a63eb44bf965ba3d45",
-          "version": null
+          "branch": null,
+          "revision": "16ab4d657e1ad4e77bd5f8b94af8538561643053",
+          "version": "2.14.0"
         }
       },
       {
         "package": "Yams",
         "repositoryURL": "https://github.com/jpsim/Yams.git",
         "state": {
-          "branch": "master",
-          "revision": "5fa313eae1ca127ad3c706e14c564399989cb1b1",
-          "version": null
+          "branch": null,
+          "revision": "c947a306d2e80ecb2c0859047b35c73b8e1ca27f",
+          "version": "2.0.0"
         }
       }
     ]

--- a/Package.swift
+++ b/Package.swift
@@ -1,27 +1,29 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.2
 import PackageDescription
 
 let package = Package(
     name: "toolbox",
     platforms: [
-       .macOS(.v10_15)
+       .macOS(.v10_15),
     ],
     products: [
-        .executable(name: "vapor", targets: ["Executable"])
+        .executable(name: "vapor", targets: ["Executable"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-nio.git", .branch("master")),
-        .package(url: "https://github.com/jpsim/Yams.git", .branch("master")),
-        .package(url: "https://github.com/tanner0101/mustache.git", .branch("master")),
-        .package(url: "https://github.com/vapor/console-kit.git", .branch("master")),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
+        .package(url: "https://github.com/jpsim/Yams.git", from: "2.0.0"),
+        .package(url: "https://github.com/tanner0101/mustache.git", from: "0.1.0"),
+        .package(url: "https://github.com/vapor/console-kit.git", from: "4.0.0-rc"),
     ],
     targets: [
         .target(name: "VaporToolbox", dependencies: [
-            "ConsoleKit",
-            "Mustache",
-            "NIO",
-            "Yams"
+            .product(name: "ConsoleKit", package: "console-kit"),
+            .product(name: "Mustache", package: "mustache"),
+            .product(name: "NIO", package: "swift-nio"),
+            .product(name: "Yams", package: "Yams"),
         ]),
-        .target(name: "Executable", dependencies: ["VaporToolbox"]),
+        .target(name: "Executable", dependencies: [
+            .target(name: "VaporToolbox"),
+        ]),
     ]
 )

